frontend/openchat-client/src/state/app/stores.ts

```diff
@@ -543,6 +538,22 @@ export const userGroupSummariesStore = derived(communitiesStore, (communities) =
     }, new Map<number, UserGroupSummary>());
 });
 
+export const serverEventsStore = writable<EventWrapper<ChatEvent>[]>([], undefined, eqIfEmpty);
```

Comment:
I updated some of the equalityCheck functions, other than that only the ordering of these stores changed, so that the stores related to the `selectedChatIdStore` can all be cleared as the first subscription on that store.